src/discordtipbot/discordbot.cr

```diff
@@ -58,6 +58,10 @@ class DiscordBot
       end
     end
 
+    @bot.on_ready do
+      @log.info("#{@config.coinname_short}: #{@config.coinname_full} bot received READY")
```

Comment:
This will run on every (re-)connection. The previous logging line would run once per process lifetime. If you wanted to retain this behavior, a flag (`@started = true`, doesn't matter) is needed here.

Reply:
I actually more wanted the current behaviour I think.